config/locales/en.yml

```diff
@@ -0,0 +1,210 @@
+    clear_text: Clear Search Text
+    dates:
+      # clearing the date filter
+      clear_filter: Clear Date Filter
+      # the word "date" singular
+      date: Date
+      # label for earliest date in range
+      date_from_label: Date From
+      # label for latest date in range
+      date_to_label: Date To
+      # earliest date in range day
+      from_day: From Day
+      # earliest date in range month
+      from_month: From Month
+      # earliest date in range year
+      from_year: From Year
+      # date filter help label
+      help: Date Filter Help
+      # divider between a range of dates, such as "2012 to 2014"
+      range_divider: to
+      # hidden label for date filter dropdown
+      show_filters: Show Date Filters
+      # latest date in range day
+      to_day: To Day
+      # latest date in range month
+      to_month: To Month
+      # latest date in range year
+      to_year: To Year
+    help:
+      # section title
+      title: Search Help
+      # search help html, this may need to be recreated per language
+      # instead of translated literally due to differences with stemming and grammar
+      html: |-
+        <h4>Basic Search</h4>
+        <p>Simply enter the word you wish to find and the search engine will search for every instance of the word in the journals. For example: Fight. All instances of the use of the word fight will show up on the results page.</p>
+        <h4>Wildcard Searches</h4>
+        <p>Using an asterisk (*) will increase the odds of finding the results you are seeking. For example: Fight*. The search results will display every instance of fight, fights, fighting, etc. More than one wildcard may be used. For example: *ricar*. This search will return most references to the Aricara tribe, including Ricara, Ricares, Aricaris, Ricaries, Ricaree, Ricareis, and Ricarra. Using a question mark (?) instead of an asterisk (*) will allow you to search for a single character. For example, r?n will find all instances of ran and run, but will not find rain or ruin.</p>
+        <h4>Capitalization</h4>
+        <p>Searches are not case sensitive. For example: george will come up with the same results as George.</p>
+        <h4>Phrase Search</h4>
+        <p>Searching for a specific phrase may help narrow down the results. Rather long phrases are no problem. For example: "This white pudding we all esteem".</p>
+        <h4>Subsequent Searches</h4>
+        <p>Because of the creative spellings used by the journalists, it may be necessary to try your search multiple times. For example: P?ro*. This search brings up numerous variant spellings of the French word pirogue, "a large dugout canoe or open boat." Searching for P?*r*og?* will bring up other variant spellings. Searching for canoe or boat also may be helpful.</p>
+    filters:
+      # clearing filters (not the text search)
+      clear_all: Clear Filters
+      # the word "filter" singular
+      filter: Filter
```

Comment:
I had a lot of trouble coming up with ways to describe how this is used, since I could see it coming up in more contexts than it is in currently, but I could also put in something more specific about it being for the submit button for date filters?

Reply:
I mean, if we want to be more specific by functionality, I could put it in the search.date section?

Reply:
Hmm. It probably can apply to different situations, but I think it'd be even trickier to document multiple places a translation string is used if we tried to generalize them. I think going specific is good with something like "label for the date filters submit button"